### ISSUE TITLE
Could you provide a model based on Xception?

### ISSUE THREAD

**Issue by M-SunRise** [sowmen/face-cutout#5]

**Comment by M-SunRise** [comment]
Another question, how can I download the pretrained models and validation_dataset.zip. Thanks for you a lot!

**Comment by sowmen** [comment]
metadata.json contains the data set paths and labels. For each image it contains the video source, original/fake label and image path. 

To use Xception model, change the "tf_efficientnet_b4_ns" to "exception" in this [line](https://github.com/sowmen/face-cutout/blob/d10ebcca19a903f51353366c4cc53b8a8dc8c5c5/train.py#L78). Everything else will remain the same.

**Comment by M-SunRise** [comment]
in the dataset.py line 74: 
`label = np.clip(label, 0.01, 1 - 0.01)`

waht does it mean? Just for skipping the mask production?

**Comment by sowmen** [comment]
This is called Label Smoothing. This is a common regularization technique. You can read more about it online.

**Comment by M-SunRise** [comment]
How did you split the FF++ to train&valid, test? Face Clustering which in your paper?
I followed the FF++ [official split](https://github.com/ondyari/FaceForensics/tree/master/dataset/splits), the result on C40 is five points lower than the paper. 
Thanks.

**Comment by GiggleLy** [comment]
How do I create metadata.json?

**Comment by ramazan793** [comment]
> How did you split the FF++ to train&valid, test? Face Clustering which in your paper? I followed the FF++ [official split](https://github.com/ondyari/FaceForensics/tree/master/dataset/splits), the result on C40 is four points lower than the paper. (The backbone is Xception. When I changed to EfficientNet B4, the result is similar to Xception) Thanks.

Hey, I have the same issue here. But I have splitted FF++ based on original actor names (FF++ provided) such that folds have non-overlapping unique actors (like in paper based on unique actors). Got average ROC_AUC 0.88 for 10 folds on EfficientNet B4.
